toc.md

```diff
@@ -24,36 +23,42 @@
 * How to contribute to OSS projects
   * Define your open source goal and strategy
   * [Establish open source guiding principles and processes](Contributions-to-existing-projects.md))
-     * Guiding principles
-     * Process for expressing company approval for contributions
-     * Contribution models
+    * Guiding principles
+    * Process for expressing company approval for contributions
+    * Contribution models
   * Build an open source metrics strategy when contributing to open source projects
-     * Goal-question-metric approach
-
+    * Goal-question-metric approach
 
 ## [Leadership Level](starting-oss-projects.md)
 
 * Why to start own OS projects? Overview of different variants (attract community, transparency, foundation-based)
-* Hot to start an OS project?
-
-  * Managing project life cycle
-
-  * Work with legal and governance aspects
-    * License
-    * CLA/DCO
-    * Code of conduct
-    * Governance (open governance, transparency about governance)
-    * Community management
-
-  * Technical considerations
-    * Setting up a repo (templates, health files, license, readme, contribution guidelines)
-    * REUSE
-    * CLA Assistant
-    * Credential scanner
-    * CII Badge
-    * Day-2-ops (CI/CD, security, license scanning, …)
-
-  * Build an open source metrics strategy when releasing to open source projects
-     * Goal-question-metric approach
+* Not to start an OS project?
+
+* Project life cycle
+  * Planning or Concept Phase
+  * Active or Development Phase
+  * Mature or Maintenance Phase
+  * Obsolete or End of Life Phase
+
+* Legal and governance considerations
+  * Which license to select
+  * Contributor License Agreement (CLA), Developer Certificate of Origin (DCO)
+
+* Community Management
+  * Governance (open governance, transparency about governance)  
```

Comment:
I would see governance as something which goes beyond community, so I wouldn't put it under community management. I tend to think that keeping it under the title "legal and governance considerations" in the order as it is in the document (license, cla, code of conduct, governance, community management) would work ok. This goes from most formal to least formal, covering the different aspects.

Reply:
I was indeed a little bit unsure where to put it. As you suggested I will put it back to the "legal and governance ..." chapter.